flex-checks/src/main/java/org/sonar/flex/checks/NonEmptyCaseWithoutBreakCheck.java

```diff
@@ -49,12 +49,32 @@ public void init() {
 
   @Override
   public void visitNode(AstNode astNode) {
-    AstNode lastAstNode = astNode.getLastChild();
-    if (lastAstNode.getFirstChild().is(FlexGrammar.STATEMENT)
-      && lastAstNode.getFirstChild().getFirstChild().isNot(FlexGrammar.BREAK_STATEMENT, FlexGrammar.RETURN_STATEMENT, FlexGrammar.THROW_STATEMENT)) {
+    int jumpStmtNumber = astNode.getDescendants(FlexGrammar.BREAK_STATEMENT, FlexGrammar.RETURN_STATEMENT, FlexGrammar.THROW_STATEMENT).size();
+
+    if (jumpStmtNumber < 2) {
```

Comment:
@vilchik-elena We are removing false positives by adding false negatives, right? (It's not an issue, we just have to be aware of it)